models/contacts.js

```diff
@@ -1,19 +1,102 @@
-// const fs = require('fs/promises')
+const fs = require("fs/promises");
+const { nanoid } = require("nanoid");
 
-const listContacts = async () => {}
+const contacts = fs.readFile("models/contacts.json");
```

Comment:
To make it cleaner maybe create 
```
const getContacts = async () => {
const contacts = fs.readFile("models/contacts.json");
return JSON.parse(await contacts);
}
```
And use it instead of  JSON.parse(await contacts);

Reply:
done